src/client/client.go

```diff
@@ -282,8 +289,13 @@ func (s *SignalClient) Init() error {
 	return nil
 }
 
+func (s *MessageMention) toString() string {
+    return fmt.Sprintf("%d:%d:%s", s.Start, s.Length, s.Author)
+}
+
 func (s *SignalClient) send(number string, message string,
-	recipients []string, base64Attachments []string, isGroup bool) (*SendResponse, error) {
+	recipients []string, base64Attachments []string, isGroup bool, mentions []MessageMention,
+	quote_timestamp *int64, quote_author *string, quote_message *string, quote_mentions []MessageMention) (*SendResponse, error) {
```

Comment:
Thanks for the changes! Just a small thing, but could you please make the variable names camelCased. Many thanks! Apart from that it looks fine.

Reply:
Well caught! Too many copy/paste...